test/index.js

```diff
@@ -131,6 +131,71 @@ suite(`${pkg.name} unit`, () => {
     assert(formattedDate.includes('Mar'))
     assert(formattedDate.includes('08:00 AM'))
   })
+
+  test('schedule parsing with timezone - UTC default', () => {
+    const schedules = ['2024-03-27T21:00:00.0Z/P2W']
+    const now = Temporal.Instant.from('2024-03-20T13:00:00.0Z')
+    const next = meetings.getNextScheduledMeeting(schedules, now)
+    const expected = Temporal.Instant.from('2024-03-27T21:00:00.0Z')
+    assert.strictEqual(next.epochMilliseconds, expected.epochMilliseconds)
+  })
+
+  test('schedule parsing with timezone - Chicago', () => {
+    const schedules = ['2024-03-27T21:00:00.0/P2W']
+    const now = Temporal.Instant.from('2024-03-20T13:00:00.0Z')
+    const next = meetings.getNextScheduledMeeting(schedules, now, 'America/Chicago')
+    // 21:00 CDT == 02:00 UTC next day
+    const expected = Temporal.Instant.from('2024-03-28T02:00:00.0Z')
```

Comment:
Any reason to still specify in UTC vs with an offset? It should make it easier to read so I don't have to verify the comment above is accurate.

Reply:
I guess just because then my brain sort of thinks it's looking at: `assert.strictEqual(true, true)` ?

Reply:
if I add an _additional_ assertion rather than changing the existing one, is the that better or worse from your perspective?